tests/phpunit/PaymentMethods/WC_Stripe_UPE_Payment_Method_Amazon_Pay_Test.php

```
@@ -76,6 +76,31 @@ public function test_supported_currencies( string $account_country, array $expec
+
+		$stripe_instance = \WC_Stripe::get_instance();
+		$initial_account = $stripe_instance->account;
+		$stripe_instance->account = $mock_account;
+
+		$supported_currencies = \WC_Stripe_UPE_Payment_Method_Amazon_Pay::get_amazon_pay_supported_currencies();
+
+		// Reset account before asserting.
+		$stripe_instance->account = $initial_account;
+
+		$this->assertEquals( $expected_currencies, $supported_currencies );
+	}
```

PHPMD (2.15.0)

[error] The method test_get_amazon_pay_supported_currencies is not named in camelCase. (undefined)

(CamelCaseMethodName)